middleware/middleware-1.3.39.md

```diff
@@ -0,0 +1,45 @@
+---
+slug: /release-notes/middleware/1.3.39
+title: Version 1.3.39
+---
+
+# fiskaltrust.Middleware 1.3.39 (Germany)
+_September 05, 2022_
+
+In this version of the Middleware,we made it possible to run the German middleware on an ARM device,moreover, we implemented an automatic retry when updating the FCC, we've also fixed a bug concerning Fail-transactions not removing the failed start transactions on the FailedStartTransaction table in the database.
+
+:::caution
+
+Version 1.3 of the Middleware is meant for the German market only, customers in Austria and France should continue to use version 1.2. We will unify these experiences in an upcoming version.
+
+:::
+
+## Feature: ARM support for the German middleware
+We now offer support for customers using ARM devices and wish to implement our German middleware. Most packages should work on ARM processors, except for these ones , because they do not offer ARM support in their software development kits:
```

Comment:
```suggestion
We now offer support for customers using ARM devices and wish to implement our German middleware. Most packages should work on ARM processors, except for these ones, because they do not offer ARM support in their software development kits:
```